### ISSUE TITLE
Error feedback when using area selection.

### ISSUE THREAD

**Issue by 5igm** [Arckenver/Nations#18]
forge-1.8.9-11.15.1.1902-1.8.9-universal
spongeforge-1.8.9-1890-4.2.0-BETA-1653
Nations-0.10

```
[00:29:03] [Server thread/ERROR] [Sponge/]: Could not pass InteractBlockEvent$Primary$Impl to org.spongepowered.mod.plugin.SpongeModPluginContainer@40b833d9
java.lang.NullPointerException
    at com.arckenver.nations.listener.GoldenAxeListener.onPlayerLeftClick(GoldenAxeListener.java:78) ~[GoldenAxeListener.class:0.10]
    at org.spongepowered.common.event.listener.PrimaryListener_GoldenAxeListener_onPlayerLeftClick95.handle(Unknown Source) ~[?:?]
    at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95) ~[RegisteredListener.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:201) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:184) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:229) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:214) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.common.SpongeImpl.postEvent(SpongeImpl.java:129) [SpongeImpl.class:1.8.9-1890-4.2.0-BETA-1653]
    at net.minecraft.network.NetHandlerPlayServer.redirect$handleLeftBlockClick$0(NetHandlerPlayServer.java:515) [lm.class:?]
    at net.minecraft.network.NetHandlerPlayServer.func_147345_a(NetHandlerPlayServer.java:525) [lm.class:?]
    at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:40) [ir.class:?]
    at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:10) [ir.class:?]
    at org.spongepowered.common.network.PacketUtil.onProcessPacket(PacketUtil.java:121) [PacketUtil.class:1.8.9-1890-4.2.0-BETA-1653]
    at net.minecraft.network.PacketThreadUtil$1.redirect$onProcessPacket$0(SourceFile:39) [fh$1.class:?]
    at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [fh$1.class:?]
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_101]
    at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_101]
    at net.minecraft.util.Util.func_181617_a(SourceFile:44) [g.class:?]
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:660) [MinecraftServer.class:?]
    at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:344) [ko.class:?]
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:605) [MinecraftServer.class:?]
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:481) [MinecraftServer.class:?]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_101]
```

**Comment by 5igm** [comment]
This error showed when I used golden axe to left-click a block in the world.

**Comment by Arckenver** [comment]
Same as #14. This is fixed, but the fix will be available in next release of Nations (v0.11).
